analyzers/tests/SonarAnalyzer.UnitTest/Extensions/InterpolatedStringExpressionSyntaxExtensionsTests.cs

```diff
@@ -56,13 +56,25 @@ public void TryGetGetInterpolatedTextValue_Returns_False(string code)
         }
 
         [DataTestMethod]
-        [DataRow("var textOnly = $\"TextOnly\" ", "TextOnly")]
-        [DataRow(@"const string constantString = ""ConstantValue"";
-                 const string constantInterpolation = $""{constantString} with text.""", "ConstantValue with text.")]
-        [DataRow(@"const string constantString = ""ConstantValue"";
-                 const string constantInterpolation = $""{$""Nested {constantString}""} with text.""", "Nested ConstantValue with text.")]
-        [DataRow(@"notConstantString = ""SomeValue"";
-                 string interpolatedString = $""{notConstantString}""", "SomeValue")]
+        [DataRow(@"
+                   var textOnly = $""TextOnly"";
+                 ",
+                 "TextOnly")]
+        [DataRow(@"
+                    const string constantString = ""ConstantValue"";
+                    const string constantInterpolation = $""{constantString} with text."";
```

Comment:
`constantString` and `ConstantValue` both start with `constant` which adds a bit of confusion when reading `{constantString}` - I suggest to replace "ConstantValue" with `"Foo"` across all tests